build_sdk/apiwrappers/csharp/App.config

```diff
@@ -1,6 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
-<configuration>
```

Comment:
Why did these reappear? All code wrappers are now done from the SAM code generation menu.